nnotepad/js/index.js

```diff
@@ -1,33 +1,43 @@
 import {Util} from './util.js';
 import {NNotepad, ParseError} from './nnotepad.js';
+import * as monaco from 'monaco-editor';
```

Comment:
It looks like this reference depends on using the 'vite' server (via `npm run start` in the root); is there any chance to make this an explicit reference so the site can be tested using just the lightweight `python3 -m http.server 8888` ?

Reply:
Use browser script rather tham esm import instead, to make the both "npm run start" and "python3 -m http.server 8888" work, please note that WebNN requires secure context (+https) to work.

Reply:
Thanks! It's worth noting that "localhost" is considered a secure context, even with HTTP protocol. See https://w3c.github.io/webappsec-secure-contexts/#is-origin-trustworthy